woocommerce/wc-admin
# Add MailPoet to Installed marketing extensions

Fixes #8088

Show MailPoet in Installed marketing extensions.

### Screenshots

<img width="908" alt="Screenshot 2021-12-27 at 22 35 55" src="https://user-images.githubusercontent.com/470616/147571514-29172382-559e-4a72-8643-ea3dffa520f1.png">

### Detailed test instructions:

1. Go to **Marketing > Overview**
2. MailPoet is not shown in **Installed marketing extensions**
3. Go to **Plugins** and install but don't activate **MailPoet 3**
4. Go to **Marketing > Overview**
5. See MailPoet in **Installed marketing extensions**
6. Click **Activate**
7. Click **Finish Setup**
8. Finish MailPoet setup (fill with dummy data)
9. Go to **Marketing > Overview**
10. See MailPoet links to Docs, Support, and Settings
